pimd/pim_cmd.c

```diff
@@ -6861,12 +6870,33 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
-DEFUN (ip_pim_v6_secondary,
-       ip_pim_v6_secondary_cmd,
-       "ip pim send-v6-secondary",
-       IP_STR
-       "pim multicast routing\n"
-       "Send v6 secondary addresses\n")
+DEFPY(igmp_group_watermark, igmp_group_watermark_cmd,
+      "ip igmp watermark-warn (10-60000)$limit",
+      IP_STR IGMP_STR
+      "Configure group limit for watermark warning\n"
+      "Group count to generate watermark warning\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	pim->igmp_watermark_limit = limit;
+
+	return CMD_SUCCESS;
+}
+
+DEFPY(no_igmp_group_watermark, no_igmp_group_watermark_cmd,
```

Comment:
A second DEFPY to do the no form?  Please make 1 DEFPY

Reply:
Sorry. Missed it. Will make it a single command.

Reply:
Will add to documentation as well

Reply:
Sorry to jerk you around here, but I disagree with donald on this one, I think the split was correct.

Reply:
Split in back into 2 commands